starters/solidjs-tanstack-tailwind/src/pages/Home/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { A, NavLink } from '@solidjs/router';
+
+const Home = () => {
+  return (
+    <div class="flex flex-col gap-2 items-center text-blue-800 underline text-base">
+      <A
```

Comment:
Why did you use `A` for this link and `NavLink` for the other? My preference goes to `NavLink` simply because a component with a single letter for a name looks weird to me.

Reply:
Another reason to stay away from `A` is that it's just a typo or formatter gone haywire away from becoming `<a href="">`